TimePicker/TimePickerBase.js

```diff
@@ -236,7 +239,7 @@ const TimePickerBase = kind({
 		delete rest.rtl;
 
 		return (
-			<DateTime {...rest} css={css}>
+			<DateTime {...rest} css={css} className={classnames(className, dateTimeCss.timePicker)}>
```

Comment:
This shouldn't be necessary. The `css` override system should offer this functionality already, along with all of the below `classnames` usages, as long as the class names in the TimePicker's LESS file have the same name as the ones they're overriding.